kiosk/sample.txt

```diff
@@ -0,0 +1,132 @@
+
+> makecode-arcade-kiosk@0.1.0 prestart
+> node ./scripts/linkfonts.js
+
+Inlining fonts/Press_Start_2P/PressStart2P-Regular.ttf
+Inlining fonts/Share/Share-Regular.ttf
+
+> makecode-arcade-kiosk@0.1.0 start
+> react-scripts start
+
+(node:5418) [DEP_WEBPACK_DEV_SERVER_ON_AFTER_SETUP_MIDDLEWARE] DeprecationWarning: 'onAfterSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
+(Use `node --trace-deprecation ...` to show where the warning was created)
+(node:5418) [DEP_WEBPACK_DEV_SERVER_ON_BEFORE_SETUP_MIDDLEWARE] DeprecationWarning: 'onBeforeSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
+Starting the development server...
+
+Compiled with warnings.
+
+Failed to parse source map from '/home/srietkerk/make-code/pxt-arcade/kiosk/node_modules/src/camera.ts' file: Error: ENOENT: no such file or directory, open '/home/srietkerk/make-code/pxt-arcade/kiosk/node_modules/src/camera.ts'
+
+Failed to parse source map from '/home/srietkerk/make-code/pxt-arcade/kiosk/node_modules/src/code-decoder.ts' file: Error: ENOENT: no such file or directory, open '/home/srietkerk/make-code/pxt-arcade/kiosk/node_modules/src/code-decoder.ts'
+
```

Comment:
Probably don't want this file?

Reply:
Yeah, sorry about that. Realized that I accidentally kept that in.